data/json/uncraft/comestibles/meat.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "result": "large_bone",
+    "type": "uncraft",
+    "activity_level": "LIGHT_EXERCISE",
+    "time": "27 m",
+    "qualities": [ { "id": "CUT", "level": 2 } ],
```

Comment:
I'm unsure if cut is appropriate here. Maybe require a saw and cut it into bone pieces, or a hammer and have bone splinters?